<ui version="4.0" >
 <class>DlgCreateAssembly</class>
 <widget class="QDialog" name="DlgCreateAssembly" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create new assembly</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>New assembly</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbUseTemplate" >
        <property name="text" >
         <string>Start with built-in assembly template</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="cbSelectTemplate" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbLoadFile" >
        <property name="text" >
         <string>Load existing assembly from file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="leFilename" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbBrowse" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbUseTemplate</tabstop>
  <tabstop>cbSelectTemplate</tabstop>
  <tabstop>rbLoadFile</tabstop>
  <tabstop>leFilename</tabstop>
  <tabstop>pbBrowse</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgCreateAssembly</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgCreateAssembly</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbUseTemplate</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbSelectTemplate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>49</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>81</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLoadFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>leFilename</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>56</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>71</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLoadFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbBrowse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>275</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
